src/XCCDF/item.h

```diff
@@ -101,6 +101,8 @@ struct xccdf_rule_item {
 	struct oscap_list *checks;
 	struct oscap_list *fixes;
 	struct oscap_list *fixtexts;
+	
+	char *stig_rule_id;
```

Comment:
isn't this already stored in `idents`? I don't understand the point of all the extra logic.